### ISSUE TITLE
stage 1 improvement?

### ISSUE THREAD

**Issue by AndrewTru** [tygari/EVO-Idle#5]
I like that we are learning how the cell parts work together, I think it may be a good idea to add a evolution gate.
The idea is that there's a gate with the two paths (Plant/animal) and it shows just a percentage.
If your cell goes out now it will have a _% chance of making it to stage 3.
This can be called it's difficulty curve or something like it.
Doable?
(Not an issue per say just an idea.)

**Comment by tygari** [comment]
I don't comprehend what your asking.

Stage 3 is already in game.
In this stage a players colony of cells evolves into a worm.
The conditions to reach stage 3 are a bit high because the players don't have access to everything intended like Xcross Evolution (being released in a few days), Exotic Evolution, and Event Evolution systems.
When I finish adding these I will write stage 4 Fish.  Then lower the difficulty to reach stage 3.
In stage 3 a player has to finish evolving any body stats not evolved in stage 2. 
A choice evolution is the player chooses to become a carnivore or a herbivore.
Then the rules of the game drastically change as the player knows.
There are also 2 more choice evolutions in stage 3.
Skeletal type. Internal or external.
Stage 3 boost.  Which gives an unusual effect.  A player can choose 1 of 4.
Boost increases the max of a combat talent by 5 and applies that talent score to a non talent effect.

Though it is encouraged mechanics wise to follow nature's path of evolutions.  It is not required.  Players are free to make fantastic creatures.  This will become more pronounced with the release of stages 4 fish, 5 amphibian, and 6 reptile.

**Comment by tygari** [comment]
PS best to contact me through Kongregate Forum with design questions and suggestions.
The game is on Kongregate.

**Comment by AndrewTru** [comment]
good point, my idea is more around informing the player the chance of your cell even getting to the next stage.
so cell to worm to fish. if I only get 10 ATP at abboundent food then I may not live long to get to worm from cell and again if I only have 5 Cali then that 5 ATP at abboundent food per sec.
ask yourself 'will the cell be able to live in that configuration or will I need more of something?'
thanks for the info.

**Comment by tygari** [comment]
That would be nice but not doable.

Many evolutions are no required to move from stage 1 cell to stage 2 colony.
They are presented but not required.
A player does not need cillia or flagellum to unlock stage 2.  But reaching stage 2 without them is pretty much impossible.

Even after a player unlocks stage 2 in the evolution options there are still many evolutions a player can try to acquire that are further along the stage 1 evolution tree.
These evolutions are not available once a player goes to stage 2.

It is not a race to proceed up evolution stages as fast as one can.  Staying in stage 1 longer means more development and more evolutions to acquire.  But this is countered by it being harder to proceed further with the evolution point limit.  A player NEEDS to reincarnate to gain Rec points.  Rec points add to a players evolution points without increasing evolution cost.
